homeassistant/components/home_connect/quality_scale.yaml

```diff
@@ -67,27 +52,30 @@ rules:
   docs-data-update: todo
   docs-examples: todo
   docs-known-limitations: todo
-  docs-supported-devices: todo
-  docs-supported-functions: done
+  docs-supported-devices:
+    status: exempt
+    comment: |
+      There is not information about supported or unsupported devices.
```

Comment:
Maybe we can just add this link to the docs?

https://www.home-connect.com/global/smart-home-appliances

Reply:
https://github.com/home-assistant/home-assistant.io/pull/37676